src/benchmarks/gc/scenarios/CrankConfiguration.yaml

```
@@ -0,0 +1,35 @@
+imports:
+  - https://github.com/aspnet/Benchmarks/blob/main/scenarios/aspnet.profiles.yml?raw=true
+
+jobs:
+  gcperfsim:
+    source:
+      repository: https://github.com/dotnet/performance
+      branchOrCommit: main
+      project: src/benchmarks/gc/src/exec/GCPerfSim/GCPerfSim.csproj
+    waitForExit: true
+    variables:
```

```suggestion
    readyStateText: GCPerfSim
    variables:
```

To let the agent automatically detect when the application has started

---

Note that this is because the app is outputting something like `GCPerfSim /tmp/benchmarks-agent/benchmarks-server-1/5pqcst5g.swz/performance/src/benchmarks/gc/src/exec/GCPerfSim/published/GCPerfSim.dll -tc 36 -tagb 540 -tlgb...`